lib/collection/script.js

```diff
@@ -17,11 +17,42 @@ _.inherit((
      * @param {Object} options
      */
     Script = function PostmanScript (options) {
+        // this constructor is intended to inherit and as such the super constructor is required to be executed
+        Script.super_.apply(this, arguments);
+
+        options && this.update(options);
+    }), Property);
+
+_.assign(Script.prototype, /** @lends Script.prototype */ {
+    /**
+     * Defines whether this property instances requires an id
+     * @private
+     * @readOnly
+     * @type {Boolean}
+     */
+    _postman_propertyRequiresId: true,
+
+    /**
+     * Converts the script lines array to a single source string.
+     *
+     * @returns {String}
+     */
+    toSource: function () {
+        return this.exec ? this.exec.join('\n') : undefined;
+    },
+
+    /**
+     * Updates the properties of a Script.
+     *
+     * @param {Object} [options]
+     * @param {String} [options.type] Script type
+     * @param {String} [options.src] Script source url
+     * @param {Array<String>|String} [options.exec] Script to execute
```

Comment:
`Array.<String>` or `String[]`

Reply:
Changing to `String[]`, but `Array<String>` is valid too.

Reply:
There's supposed to be a dot in between: http://usejsdoc.org/tags-type.html